integration/test-integration/pom.xml

```
@@ -36,6 +36,7 @@
                 <version>${maven.surefire.plugin.version}</version>
                 <configuration>
                     <systemPropertyVariables>
+                        <cc_root_dir>${project.basedir}${file.separator}..${file.separator}..</cc_root_dir>
```

```suggestion
                        < cc_root_dir >${project.parent.parent.basedir}</cc_root_dir >
```
Ref: https://stackoverflow.com/questions/1012402/maven2-property-that-indicates-the-parent-directory

---

I tried this but it didn't work. Had to go with `/../..`

---

removed this line since we are not referring to the ca cert in the root folder anymore